etsin_finder/frontend/js/components/general/navigation/index.jsx

```diff
@@ -35,7 +36,7 @@ const Navi = ({ routes }) => {
         }
       }}
       onClick={() => {
-        announce(translate('changepage', { page: translate(route.label) }))
+        Accessibility.announce(translate('changepage', { page: translate(route.label) }))
```

Comment:
I noticed the same bug, it's because how `this` works in javascript. It's better not to import the stores directly. I'd revert the changes in this file and edit the constructor `accessibility.js` like this:
```
class Accessibility {
  constructor() {
    makeObservable(this)
    this.announce = this.announce.bind(this)
  }
```

Reply:
Great suggestion. Resolved with 3ef8c9e and 15761bf